styles/react.dev/catppuccin.user.less

```
@@ -0,0 +1,442 @@
+    }
+
+    .hover\:bg-card:hover {
+      background-color: @crust;
+    }
+
+    //code stylizing
+
+    a > code {
+      color: @accent !important;
+    }
+
+    .console-block code {
+      background-color: @surface1 !important;
+      color: @red !important;
+    }
+
+    .sp-wrapper {
+      --sp-colors-accent: @accent;
+      --sp-colors-clickable: @flamingo;
+      --sp-colors-disabled: @text;
+      --sp-colors-error: @red;
+      --sp-colors-error-surface: @maroon;
+      --sp-colors-surface1: @surface0;
+      --sp-colors-surface2: @surface1;
+      --sp-syntax-color-plain: @text;
+      --sp-syntax-color-comment: @overlay2;
+      --sp-syntax-color-keyword: @mauve;
+      --sp-syntax-color-tag: @blue;
+      --sp-syntax-color-punctuation: @overlay2;
+      --sp-syntax-color-definition: @yellow;
+      --sp-syntax-color-property: @blue;
+      --sp-syntax-color-static: @peach;
+      --sp-syntax-color-string: @green;
+    }
+
+    //border styling
+
+    .border-blue-40 {
+      border-color: @blue;
+    }
+
+    .border-yellow-40 {
+      border-color: @yellow;
+    }
+
+    .border-purple-40 {
+      border-color: @lavender;
+    }
+
+    .dark\:border-purple-30, .border-purple-30 {
+      border-color: @mauve;
+    }
+
+    .border-green-40 {
+      border-color: @green;
+    }
+
+    .border-gray-300, .bg-gray-90 {
+      border-color: @overlay2;
+    }
+
+    .border-link:hover {
+      border-color: @accent;
+    }
+
+    .dark\:border-link-dark {
+      border-color: @accent;
+    }
+
+    //backgrounds styling
+
+    .bg-card-dark,
+    .bg-card {
+      background-color: @surface0;
+    }
+
+    .bg-highlight {
+      background-color: @surface0;
+    }
+
+    .bg-highlight-dark {
+      background-color: @surface0;
+    }
+
+    .bg-github-highlight {
+      background-color: @surface0;
+    }
+
+    .bg-link {
+      color: @surface0;
+      background-color: @accent;
+    }
+
+    .bg-gradient-left,
+    .bg-gradient-left-dark {
+      background-image: conic-gradient(
+        from 90deg at -10% 100%,
+        @mantle 0deg,
+        @base 90deg,
+        @crust 1turn
+      );
+    }
+
+    .bg-gray-3,
+    .bg-gray-5 {
+      background-color: @crust;
+    }
+
+    .bg-gray-40\/5,
+    .bg-gray-60\/5 {
+      background-color: @overlay1;
+    }
+
+    .bg-gray-40\/10,
+    .bg-gray-60\/10 {
+      background-color: @overlay0;
+    }
+    .bg-gray-30,
+    .bg-gray-300,
+    .bg-gray-70,
+    .bg-gray-700 {
+      background-color: @surface0;
+    }
+
+    .bg-gray-80,
+    .bg-gray-200,
+    .bg-gray-95 {
+      background-color: @mantle;
+    }
+
+    .bg-black {
+      background-color: @base;
+    }
+
+    #color-fragment-style(@divColor) {
+      background-color: fade(@divColor, 40%);
+      color: @text;
+
+      p {
+        color: @text;
+      }
+
+      button {
+        background-color: darken(@mauve, 13%) !important;
+
+        svg {
+          color: @text !important;
+        }
+
+      }
```

I'm pretty sure that colors should not be darkened by more than 10%.

```suggestion
      button {
        color: @base !important;
        background-color: @divColor !important;

        svg {
          color: @base !important;
        }
      }
```